### ISSUE TITLE
Is it possible to disable user signup?

### ISSUE THREAD

**Issue by ashmckenzie** [huginn/huginn#2824]
I can't see a way to disable user signup?  As a workaround I've set an absurdly long invitation code but it would be nice to be able to completely disable the ability 🙂

**Comment by dsander** [comment]
You are right, that is the current "solution", I would not object a PR that improves it 😄
